niceml/dlframeworks/tensorflow/models/mlp.py

```diff
@@ -22,12 +24,14 @@ def __init__(
         final_activation: str = "linear",
         do_summary: bool = True,
     ):
+        """Constructor"""
```

Comment:
Please add more details. I think it's fine if you use the class docstring in combination with a description of the attributes.